src/components/forms/email-template-form.js

```diff
@@ -228,8 +243,8 @@ const EmailTemplateForm = ({ entity, match, errors, clients, preview, templateLo
         };
     });
 
-
     const email_clients_ddl = clients ? clients.map(cli => ({ label: cli.name, value: cli.id })) : [];
+    const versions_ddl = stateEntity.versions ? stateEntity.versions.map(v => ({ label: v.last_modified, value: v.sha })) : [];
```

Comment:
label per request should follow this format
last_modfied - sha1 - commit message
also it should have a link to open the html_url attribute to redirect to github ( opening on a new tab )